### ISSUE TITLE
Validate BiBTeX output with a LaTeX, BiBTeX, LaTeX cycle that runs automatically as part of the GitHub Actions

### ISSUE THREAD

**Issue by simsong** [sciunto-org/python-bibtexparser#313]
Validate BiBTeX output with a LaTeX, BiBTeX, LaTeX cycle that runs automatically as part of the GitHub Actions.

This is the only real way to validate that everything works.

**Comment by MiWeiss** [comment]
Assigned the issue to @simsong  (discussed in another thread). Thanks for taking this over. This looks like a great reliability smokescreen!

Just to be sure we're talking about the same thing: You intend to make sure that files written using the writer are valid, right? I guess that leaves some wiggle-room w.r.t. what's considered a bug, based on the input you feed it (garbage in / garbage out). It also raises the question of how we should handle potential problems identified by the workflow: I guess we'd have to open issues for each one of them and, until all of them are fixed, not enforce the new workflow's pass (as otherwise, no PRs could be merged until all of this is fixed). 

Looking forward to the contributions, I think this might be very useful. 

Note: These tests would have to run in a separate workflow (not in the unit test one). They probably run slower and need more and different dependencies.

**Comment by MiWeiss** [comment]
@simsong V2 (beta) which fixes many of the issues of v1 is now available on the main branch. 

Are you still interested in contributing a test suite as described in this issue? If so, I think it would make sense to port it directly to v2. Otherwise, I will close this issue as stale.

**Comment by simsong** [comment]
@MiWeiss Yes, I remain interested in contributing a test suite. Test suites are important. it could also be integrated with GitHub Actions, which I could also do.

**Comment by MiWeiss** [comment]
Great, I am happy to keep this open then :-) Thanks!

**Comment by simsong** [comment]
So it looks like I should check out v2 beta. I'll do that and put this on my task list. Thanks!
